tools/defillama-cli/defillama-cli.py

```
@@ -0,0 +1,790 @@
+
+    if args.json:
+        result = api_get("/swap", {"chain": chain, "from": token_in, "to": token_out, "amount": amount})
+        print(json.dumps(result, indent=2))
+        return
+
+    data = api_get("/swap", {"chain": chain, "from": token_in, "to": token_out, "amount": amount})
+    if isinstance(data, dict) and "error" in data:
+        print(red(f"Quote error: {data['error']}"))
+        return
+
+    print(bold(f"\n=== Swap Quote ===\n"))
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)